src/test/groovy/com/causecode/fileuploader/DefaultUFileTemporaryUrlRenewerServiceSpec.groovy

```diff
@@ -0,0 +1,38 @@
+package com.causecode.fileuploader
+
+import com.causecode.fileuploader.cdn.google.GoogleCDNFileUploaderImpl
+import com.causecode.fileuploader.util.renewer.DefaultUFileTemporaryUrlRenewer
+import grails.test.mixin.TestFor
+import spock.lang.Specification
+
+/**
+ * Unit tests for {@link DefaultUFileTemporaryUrlRenewer}
+ *
+ * @author Milan Savaliya
+ * @since 3.1.3
+ */
+@TestFor(UFileTemporaryUrlRenewerService)
+class DefaultUFileTemporaryUrlRenewerServiceSpec extends Specification {
```

Comment:
Service name does not include `Default`. The test class name should be `UFileTemporaryUrlRenewerServiceSpec`.

Reply:
Fixed.